requirements.txt

```diff
@@ -31,7 +31,7 @@ pytest-mock>=1.1,<2.0
 # PEP 484
 # License: PSF
 # Upstream url: https://github.com/python/typing
-typing==3.6.4
+typing==3.6.4;python_version<"3.5"
```

Comment:
if we default to use python36, or python37, do we still need this dep? I thought it come with default?